torch/_library/utils.py

```diff
@@ -78,24 +78,29 @@ def is_functional_schema(schema: Any) -> bool:
     - it has at least one return
     """
 
+    def is_functional(schema):
+        if schema.is_mutable:
+            return False
+        rets = schema.returns
+        is_non_mutating_view = len(rets) > 0 and any(
+            r.alias_info is not None and not r.alias_info.is_write for r in rets
+        )
+        if is_non_mutating_view:
+            return False
+        if not schema.returns:
+            return False
```

Comment:
this is a weird schema: not inplace and no output haha

Reply:
this usually just means the user did write an inplace op but forgot to label it as inplace.